docs/content/en/blog/releases/v0.26.0.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Release v0.26.0"
+linkTitle: "Release v0.26.0"
+date: 2022-02-21
+description: >
+ Release v0.26.0
+---
+
+## Changelog since v0.25.1
+
+### Breaking changes
+
+The environment concept has been removed.
+
+It was deprecated at [v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0) and from this version it will be no longer supported. Please check this [guide](https://pipecd.dev/blog/2022/02/10/february-2022-update/#environment-and-deployment-configuration-file-were-deprecated) to switch to use Label instead.
```

Comment:
```suggestion
It was marked as deprecated from [v0.24.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.24.0), and from this version, it will be no longer supported. Please check this [guide](https://pipecd.dev/blog/2022/02/10/february-2022-update/#environment-and-deployment-configuration-file-were-deprecated) to switch to use Label instead.
```
nits